common/json.c

```diff
@@ -216,3 +216,54 @@ jsmntok_t *json_parse_input(const char *input, int len, bool *valid)
 
 	return toks;
 }
+
+void json_tok_print(const char *buffer, const jsmntok_t *params)
+{
+	const jsmntok_t *first = params;
+	const jsmntok_t *last = json_next(params);
+	printf("size: %d, count: %ld\n", params->size, last - first);
+	while (first != last) {
+		printf("%ld. %.*s, %d\n", first - params,
+		        first->end - first->start, buffer + first->start,
```

Comment:
"%td" again I think for first param.  We should add a `jsmntype_to_string` for the type printing too, which will be prettier :)